src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,9 @@
+* {
```

Comment:
use '*' only for box-sizing

Reply:
agree. * should be used only for box-sizing